### ISSUE TITLE
False Accessibility - Color Contrast test when using white offscreen text.

### ISSUE THREAD

**Issue by Snapstromegon** [GoogleChrome/lighthouse#3143]
I'm currently working on a wordclock and while testing it, I found out, that the Accessibility test marks some of my texts as not readable, but they are white (#ffffff) on black background (#000000).

During the test exactly those elements are getting offscreen (because it stays in the mobile view and my clock makes a square using vh) and get marked as not readable.

If this isn't a problem and I made a mistake I would like to hear what I've done wrong.

Cheers Guys,
You're great!

**Comment by patrickhulce** [comment]
@Snapstromegon thanks for the report! Is this reproducible when testing with [axe](https://github.com/dequelabs/axe-core) alone? If so, they'll probably be able to help you out more by filing an issue over there. If it's only when using axe in Lighthouse we can take another look.

**Comment by Snapstromegon** [comment]
@patrickhulce the problem is reproducable with axe alone, but only if you manually switch do the mobile view - so in my case the problem would't exist, if the Lighthouse tool switches back to the desktop view.

Nevertheless I'll fiel an issue for the axe team.

**Comment by brendankenny** [comment]
Closing on this side. We'll pick it up when/if axe updates and we re-roll the dep.
